go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.3.1
 	github.com/blang/semver/v4 v4.0.0
 	github.com/containerd/containerd v1.7.26
-	github.com/containers/image/v5 v5.33.1
+	github.com/containers/image/v5 v5.34.0
```

Comment:
@anik120 the problem here is: github.com/containers/image/v5 v5.34.0
See: https://github.com/operator-framework/operator-controller/pull/1692